server/infra/gormrepo/pullrequests.go

```diff
@@ -0,0 +1,29 @@
+package gormrepo
```

Comment:
why in an infra folder ?  you can have the implem in the same folder as the interface isn't ?